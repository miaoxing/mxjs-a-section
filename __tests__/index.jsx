import { Section } from '../';
import { render } from '@testing-library/react';
import { ConfigProvider } from 'antd';

describe('Section', () => {
  test('renders correctly', () => {
    const { container } = render(
      <ConfigProvider theme={{ hashed: false }}>
        <Section>foo</Section>
      </ConfigProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
