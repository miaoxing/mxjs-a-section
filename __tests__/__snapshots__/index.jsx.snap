// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
<div>
  <div
    class="ant-card ant-card-bordered mb-6 [&_.ant-card-body>:last-child]:mb-0"
  >
    <div
      class="ant-card-body"
    >
      foo
    </div>
  </div>
</div>
`;
